api/src/main/java/com/khali/api3/services/MembersService.java

```diff
@@ -49,7 +75,7 @@ public List<Member> getMembersByRC(ResultCenter resultCenter){
 
     public Member insertMember(User user, ResultCenter resultCenter){
         MemberPrimaryData memberPK = new MemberPrimaryData(user, resultCenter);
-        Member member = new Member(memberPK);
+        Member member = new Member(memberPK, true, null);
```

Comment:
não é necessário passar explicitamente todos os valores do Vconstrutor de `Member` pois o mesmo contém a `@NoArgsConstructor`
```suggestion
        Member member = new Member();
        member.setMemberPK(memberPK);
```
dessa forma, criamos o `Member` com todos os campos com valor `null` e apenas associamos um valor ao `memberPK`